src/renderers/webgl/WebGLProgram.js

```diff
@@ -797,9 +797,6 @@ function WebGLProgram( renderer, cacheKey, parameters ) {
 
 	// clean up
 
-	gl.detachShader( program, glVertexShader );
-	gl.detachShader( program, glFragmentShader );
```

Comment:
Instead, maybe just comment out these lines and add a comment?

Reply:
Maybe there's no point in that. We might change the entire library to use WebGPU before we reach a point where people would be ok with us dropping support for iOS 10 or lower.

Reply:
I think @WestLangley's idea makes sense.
That way if any other contributor tries to implement it again they'll realize we already tried.

Reply:
:heavy_check_mark: